src/Microsoft.OData.Edm/Library/EdmModelBase.cs

```diff
@@ -48,6 +49,11 @@ protected EdmModelBase(IEnumerable<IEdmModel> referencedModels, IEdmDirectValueA
                 this.referencedEdmModels.Add(CapabilitiesVocabularyModel.Instance);
             }
 
+            if (CommunityVocabularyModel.Instance != null)

```

Comment:
We need to add another `if` clause, if there is a new community vocabulary.  This does not seem to be right.